packages/server/graphql/public/typeDefs/Search.graphql

```diff
@@ -0,0 +1,54 @@
+union SearchResultObject = Page | Discussion | ReflectTemplate | PokerTemplate | FixedActivity
```

Comment:
+1 nit: break these different types into their own files. Larger files like this are great when you can say these all belong under `Search`, but as concepts evolve, entities have multiple parents.
e.g. Where would we stick OrganizaitonUser? under `Organization.graphql` or `User.graphql`?

Reply:
-1  More descriptions, please!